projects/storefrontlib/src/cms-structure/services/feature-modules.service.ts

```diff
@@ -177,10 +177,12 @@ export class FeatureModulesService implements OnDestroy {
             moduleRef.injector
           );
 
-          // extract cms components configuration from feature config
-          for (const componentType of featureInstance.cmsComponents) {
-            featureInstance.componentsMappings[componentType] =
-              resolvedConfiguration.cmsComponents[componentType];
+          // extract cms components configuration from feature config, if any
+          if (!!featureInstance.cmsComponents) {
```

Comment:
Some modules, like asm, could be lazy loaded by other triggers than cms components.  In those cases, they might have no cms components defined as triggers in the feature configuration.